js/demo/lazy-loading.js

```diff
@@ -109,4 +96,4 @@ var EffecktLazyLoading = {
 
 };
 
-EffecktLazyLoading.init();
\ No newline at end of file
+EffecktLazyLoading.init(window);
```

Comment:
Let's be consistent with the [other demo page](https://github.com/h5bp/Effeckt.css/blob/master/js/demo/demo.js) and not pass this for the time being, or pass it in for both. It isn't really needed IMHO.